Makefile

```diff
@@ -134,17 +139,32 @@ $(foreach p,$(REACTION_NETWORKS),$(eval $(call network-remove-template,$(p))))
 network-remove: $(foreach p,$(REACTION_NETWORKS),network-remove-$(p))
 
 ###############################################################################
-### Github cloning
+### Git cloning
 ###############################################################################
 define git-clone-template
-$(1):
-	@git clone "git@github.com:reactioncommerce/$(1).git"
+$(2):
+	git clone "$(1)/$(2).git"
```

Comment:
I think `$1` should be the full `.git` path. It's possible that we'd want, or end up with, a local folder name that doesn't match the remote, even though we'll keep them the same by convention.

Reply:
OK I'll have to add some more basename processing to that then.